samples/subsys/ipc/ipc_service/icmsg/remote/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
This is not allowed upstream.

@emob-nordic Can you please send a PR to fix this since you have signed off on this?

Reply:
Will do by the end of week.

Reply:
Thanks!

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/54491